quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/BuildCommandDistTest.java

```diff
@@ -48,6 +49,7 @@ void resetConfig(LaunchResult result) {
 
     @Test
     @Launch({ "--profile=dev", "build" })
+    @WithLegacyStoreOnly
```

Comment:
For `build` it should be working, right? 

@andre-nascimento-rh Most likely this is the spot when the `--storage` is not found. As you always add the `storage` at the first index. Needs to be specified after `build`. See [here](https://github.com/keycloak/keycloak/pull/14713/files?diff=unified&w=0#diff-aa081e5f62b9bbde53e18c058aae57608573561e504dedd279b1737badba3f64R120)

Reply:
Solved. Thanks.